### ISSUE TITLE
Uninitialized weights: `extra_msa_stack.blocks.3. msa_att_row` and related for OpenFold Multimer

### ISSUE THREAD

**Issue by eswan01** [aqlaboratory/openfold#538]
Hi, I've been running some folding with OpenFold Multimer using custom inputs, and have found some strange discrepancies between OpenFold outputs and AlphaFold outputs. In the process of tracking this down, I checked to ensure that all the weights are being properly loaded into the model, and found:

```
from openfold.config import model_config
from openfold.model.model import AlphaFold
from openfold.utils.import_weights import import_jax_weights_

cfg = model_config("model_1_multimer_v3")

model = AlphaFold(cfg)

import_jax_weights_(model, "<my_path>/params_model_1_multimer_v3.npz", version="model_1_multimer_v3")

for name, param in model.named_parameters():
    if torch.sum(param) == 0:
        print(f"Uninitialized weight: {name}, shape: {param.shape}")
```
```
=== OUTPUT ===
Uninitialized weight: extra_msa_stack.blocks.3.msa_att_row.layer_norm_m.bias, shape: torch.Size([64])
Uninitialized weight: extra_msa_stack.blocks.3.msa_att_row.layer_norm_z.bias, shape: torch.Size([128])
Uninitialized weight: extra_msa_stack.blocks.3.msa_att_row.mha.linear_o.weight, shape: torch.Size([64, 64])
Uninitialized weight: extra_msa_stack.blocks.3.msa_att_row.mha.linear_o.bias, shape: torch.Size([64])
Uninitialized weight: extra_msa_stack.blocks.3.msa_att_row.mha.linear_g.weight, shape: torch.Size([64, 64])
Uninitialized weight: extra_msa_stack.blocks.3.msa_transition.layer_norm.bias, shape: torch.Size([64])
Uninitialized weight: extra_msa_stack.blocks.3.msa_transition.linear_1.bias, shape: torch.Size([256])
Uninitialized weight: extra_msa_stack.blocks.3.msa_transition.linear_2.weight, shape: torch.Size([64, 256])
Uninitialized weight: extra_msa_stack.blocks.3.msa_transition.linear_2.bias, shape: torch.Size([64])
Uninitialized weight: extra_msa_stack.blocks.3.msa_att_col.layer_norm_m.bias, shape: torch.Size([64])
Uninitialized weight: extra_msa_stack.blocks.3.msa_att_col.global_attention.linear_g.weight, shape: torch.Size([64, 64])
Uninitialized weight: extra_msa_stack.blocks.3.msa_att_col.global_attention.linear_o.weight, shape: torch.Size([64, 64])
Uninitialized weight: extra_msa_stack.blocks.3.msa_att_col.global_attention.linear_o.bias, shape: torch.Size([64])
```

Before I spend more time exploring other rabbit holes, I wanted to check whether this is intentional behavior. The same **does not occur** when I import the monomer weights.

**Comment by tsjain** [comment]
Not sure, but this might have to do with the outer product mean for the extra_msa features being moved to the beginning in the evoformer block in Multimer (section 7.7 Architectural Modifications - AF Multimer paper).

In combination with Section 1.7.2 (Unclustered MSA Stack - Original AF paper) which states that the final MSA activations are unused, I think that the parameters in block.3 (last blocks) have no gradients flowing back to them.

If you look at the original AF2 multimer weights (params_model_1_multimer_v3.npz) you will see that many weight and biases for the last block are exactly 0 and 1. For example:

```
data = np.load(jax_path)
print(data['alphafold/alphafold_iteration/evoformer/extra_msa_stack/msa_column_global_attention/attention//output_b'][2])
print(data['alphafold/alphafold_iteration/evoformer/extra_msa_stack/msa_column_global_attention/attention//output_b'][3])
```

```
[ 1.28184289e-01 -2.64141142e-01 -7.57357851e-02 -2.11703330e-02
 -1.36838332e-01 -1.15181759e-01  2.19129980e-01 -4.44196601e-04
 -8.45720321e-02 -2.36421600e-02  1.58557206e-01 -1.11576542e-01
  1.00208871e-01  1.43553773e-02 -1.44288406e-01  2.24872366e-01
  6.95791328e-03 -4.00724150e-02  9.48276818e-02 -1.62258744e-04
  7.54782744e-03 -2.47061908e-01 -9.39482599e-02  1.74896017e-01
 -3.17399926e-03 -3.70870084e-02 -1.35395199e-01  2.46189207e-01
 -7.77312145e-02  1.44140705e-01  2.20363349e-01 -4.78302717e-01
  2.58077830e-01  1.17897213e-01 -1.20776489e-01  4.78612661e-01
  2.08930686e-01 -2.73930699e-01 -6.08355328e-02  9.56167281e-02
  1.17851838e-01 -6.83330968e-02  5.62093146e-02 -7.18816370e-02
  4.19841371e-02 -1.71731517e-01  8.09093043e-02 -5.52851371e-02
  1.04371257e-01 -2.03759735e-03 -2.49258295e-01 -2.04495922e-01
  1.38111666e-01  7.77580729e-03 -4.13306504e-02 -1.19575359e-01
  3.98526154e-02  1.61701933e-01 -3.67604494e-02  1.11952975e-01
  6.00760505e-02  1.72925077e-03  9.19277668e-02  2.61438210e-02]
[0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.
 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.
 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]
```
